src/app/Fake.DotNet.Fsi/Fsi.fs

```diff
@@ -279,7 +285,7 @@ module internal ExternalFsi =
                 { info.WithEnvironmentVariables defaultEnvironmentVars with
                     FileName = fsiExe
                     Arguments = args
-                    WorkingDirectory = ""
+                    WorkingDirectory = parameters.WorkingDirectory
```

Comment:
This is a bugfix as well, correct?

Reply:
Yes, changing the WorkingDirectory parameter was always ignored.